### PR TITLE
 ex_33.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Create a Magic 8 Ball game that prompts for a question and then displays either “Yes,” “No,” “Maybe,” or “Ask again later.”

Example Output
       What's your question?  Will I be rich and famous?
       Ask again later.

### CODE REVIEW

File: ex_33.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Step 1: import random
+import random
+
+# Step 2: create a list with answers and tell Python to shuffle them
+answers = ['yes', 'no', 'maybe', 'ask again later']
```
Comment (paulghaddad) [inline ex_33.py]: This can be a constant since this list will never change and is constant.

File: ex_33.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+random.shuffle(answers)
```
Comment (paulghaddad) [inline ex_33.py]: Wow, nice work here!

File: ex_33.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# Step 3: create a loop that asks user for their question and print a response 
+for answer in answers:
```
Comment (paulghaddad) [inline ex_33.py]: Perfect!

**Comment by amymhaddad** [comment]
@paulghaddad I updated this exercise!

File: ex_33.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Step 1: import random
+import random
+
+# Step 2: create constant and randomize the list
+ANSWERS = ['yes', 'no', 'maybe', 'ask again later']
```
Comment (paulghaddad) [inline ex_33.py]: Great!

**Pull request merged**